### ISSUE TITLE
Why flip the third dimension y = torch.flip(x, (3,)) for the invertible toy problem?

### ISSUE THREAD

**Issue by zuoshifan** [cetmann/iunets#14]
In **Tutorial 1: Training iUNets in Pytorch**, for the **An invertible toy problem**, when **Training the model,** the loss was computed by comparing the model output ```output = model(x)``` and ```y = torch.flip(x, (3,))``` instead of ```x```. Why the third dimension should be flipped? Thank you!

**Comment by tasptz** [comment]
To mirror the image:

> In the following, we will train an iUNet to **mirror the input** image as a warm-up.
